### PR TITLE
Added support for tail_addr instruction [ISSUE# 127]

### PR DESCRIPTION
**Pull request by mmroz**

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -601,6 +601,26 @@ jobject SILWalaInstructionVisitor::visitIndexAddrInst(IndexAddrInst *IAI) {
...
+  if(Print){
+    llvm::outs() << "\t [BASE ADDR]" << BaseVale.getOpaqueValue() << "\n";
+    llvm::outs() << "\t [INDEX ADDR]" << IndexValue.getOpaqueValue() << "\n";
+    llvm::outs() << "\t [RESULT TYPE]" << ResultType.getAsString() << "\n";
+  }
+
+  jobject BaseNode = findAndRemoveCAstNode(BaseVale.getOpaqueValue());
+  jobject IndexNode = findAndRemoveCAstNode(IndexValue.getOpaqueValue());
+
+  jobject Node = Wala->makeNode(CAstWrapper::ARRAY_REF, BaseNode , IndexNode);
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: remove space before last comma in parameter list

**Review by karimhamdanali — APPROVED** [review]
please see my comment below

**Review by karimhamdanali — CHANGES_REQUESTED** [review]
remove space before last comma in parameter list

**Comment by mmroz** [comment]
Resolved!

**Review by karimhamdanali — APPROVED** [review]
LGTM

**Pull request merged**